pkgs/development/python-modules/imbox/default.nix

```diff
@@ -0,0 +1,33 @@
+{ lib, buildPythonPackage, fetchFromGitHub, setuptools, chardet }:
+
+buildPythonPackage rec {
+  pname = "imbox";
+  version = "0.1.0";
```

Comment:
The latest upstream release is [0.9.6](https://github.com/martinrusev/imbox/releases/tag/0.9.6). Why do we start with 0.1.0 which was never tagged by upstream?

Reply:
Sorry